x-pack/packages/security-solution/features/src/cases/v2_features/kibana_sub_features.ts

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import type { SubFeatureConfig } from '@kbn/features-plugin/common';
+import { CasesSubFeatureId } from '../../product_features_keys';
+import { APP_ID } from '../../constants';
+import type { CasesFeatureParams } from '../types';
+
+/**
+ * Sub-features that will always be available for Security Cases
+ * regardless of the product type.
+ */
+export const getCasesBaseKibanaSubFeatureIdsV2 = (): CasesSubFeatureId[] => [
+  CasesSubFeatureId.deleteCases,
+  CasesSubFeatureId.casesSettings,
+  CasesSubFeatureId.createComment,
+  CasesSubFeatureId.reopenCase,
+];
+
+/**
+ * Defines all the Security Solution Cases subFeatures available.
+ * The order of the subFeatures is the order they will be displayed
+ */
+export const getCasesSubFeaturesMapV2 = ({
+  uiCapabilities,
+  apiTags,
+  savedObjects,
+}: CasesFeatureParams) => {
+  const deleteCasesSubFeature: SubFeatureConfig = {
+    name: i18n.translate('securitySolutionPackages.features.featureRegistry.deleteSubFeatureName', {
+      defaultMessage: 'Delete',
+    }),
+    privilegeGroups: [
+      {
+        groupType: 'independent',
+        privileges: [
+          {
+            api: apiTags.delete,
+            id: 'cases_delete_v2',
```

Comment:
@cnasikas any thoughts on what we want the versioning naming pattern to be? Just defaulted to `_v2` to keep it simple, but we should ideally start to set some kind of consistent standard...